KarhooUISDK/CommonUI/BottomSheet/KarhooBottomSheetViewModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BottomSheetMPV.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 03.11.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+protocol BottomSheetViewModel {
+    var title: String { get set }
+    var cornerRadius: CGFloat { get set }
+    var backgroundColor: Color { get set }
+    
+    func dismiss()
+}
+
+class KarhooBottomSheetViewModel: BottomSheetViewModel {
+    var title: String = ""
+    var backgroundColor: Color
+    var cornerRadius: CGFloat
```

Comment:
Why we need cornerRadius in viewModel? Do we need to customize it?

Reply:
I'd like to have the value in the viewmodel as a summary of values that can be customisable basically. I don't expect the values to change soon, but if they do (or we decide to use this bottom sheet for other scenarios than the ones we agreed upon already for booking confirmation and checkout) then we'll be able to easily customise it without touching the swiftui code anymore.